action/kafkaProduce.py

```diff
@@ -51,19 +50,21 @@ def validateParams(params):
         return (False, "You must supply all of the following parameters: {}".format(', '.join(missingParams)))
 
     if 'base64DecodeValue' in params and params['base64DecodeValue'] == True:
-        decodedValue = params['value'].decode('base64').strip()
-        if len(decodedValue) == 0:
+        try:
+            validatedParams['value'] = base64.b64decode(params['value']).decode('utf-8')
+
+            if len(validatedParams['value']) == 0:
+                raise Exception
+        except:
```

Comment:
It's typical in Python to be explicit in catching and throwing Exceptions.  Here it could be `binascii.Error`, `KeyError`, or decoding error.    Here you're also raising an Exception in the same block you're catching it.  I would split this block into try/catch for decoding, and separate the conditional check for the key being empty.  I think it would be more readable/clear that way.